promscale/distributed-tracing.md

```diff
@@ -0,0 +1,284 @@
+    endpoint: "tobs-promscale-connector.default.svc.cluster.local:9202"
+    insecure: true
+
+processors:
+  batch:
+    send_batch_size: 4
+    send_batch_max_size: 8
+
+service:
+  pipelines:
+    traces:
+      receivers: [jaeger, otlp, zipkin]
+      exporters: [otlp]
+      processors: [batch]
+```
+
+## Instrumentation
+When you have the OpenTelemetry Collector set up, you can instrument the service
+for your preferred tracing receiver.
+
+### Instrument OpenTelemetry
+To instrument OpenTelemetry, configure the OpenTelemetry SDK to export your
+traces using the OTLP exporter to the OpenTelemetry Collector OTLP receiver.
+Alternatively, you can instrument the OTLP exporter to Promscale's native OTLP
+ingest endpoint.
+
+The OTLP receiver supports both gRPC and HTTP. Promscale supports gRPC only. We
+recommend using gRPC.
+
+The OpenTelemetry Collector OTLP receiver listens on port 4317 for
+gRPC and 4318 for HTTP connections.
+
+To use gRPC, configure the OTLP exporter to send data to
+`<opentelemetry-collector-host>:4317`. If you deployed a full observability
+stack with tobs, use
+`tobs-opentelemetry-collector-collector.default.svc.cluster.local:4317`
+
+The OTLP ingest endpoint listens to gRPC connections on the address you specify
+with the `otlp-grpc-server-listen-address` parameter. By default, Promscale
+listens on port 9202, so in most cases, you can point the OTLP exporter to
+`<promscale-connector-host>:9202`. If you deployed a full observability stack
+with tobs, use `tobs-promscale-connector.default.svc.cluster.local:9202`.
+
+### Instrument Jaeger
+To instrument Jaeger, configure the Jaeger agent to send your traces to the
+OpenTelemetry Collector by starting the receiver with the
+[reporter.grpc.host.port parameter][reporter-grpc-host-port]. Set `host:port` to
+the port location where  the
+[OpenTelemetry Collector Jaeger Receiver][jaeger-receiver] is listening for
+connections. By default, the receiver listens for gRPC connections on port
+14250, so in most cases you can point the Jaeger agent to
+`<opentelemetry-collector-host>:14250`. If you deployed a full observability
+stack with tobs, use
+`tobs-opentelemetry-collector.default.svc.cluster.local:14250`.
+
+### Instrument Zipkin
+To instrument Zipkin, configure the Zipkin transport you are using to send
+traces to the [OpenTelemetry Collector Zipkin Receiver][zipkin-receiver]
+endpoint with `<opentelemetry-collector-host>:9411`. Tobs does not currently
+configure the OpenTelemetry Collector to ingest Zipkin traces.
+
+<!--- Consider starting a new sub-page for visualization. --LKB 20220202-->
+## Visualize traces with tobs
+If you used tobs to deploy Promscale and all its components, you can access
+Grafana and Jaeger to visualize your traces, and you do not need to do any
+further configuration steps.
+
+<procedure>
+
+### Visualizing traces in Grafana
+1.  Set up the Grafana user interface:
+    ```bash
+    tobs grafana get-password
+    tobs grafana port-forward
```

Comment:
How do we format two separate commands run after each other? Do we prefer putting them in the same code block or separate code blocks? 

I lean towards two code blocks because it makes it clearer they're separate and also makes copying easier, but it's not a strong preference

Reply:
Yeah, I think they should be two steps. Good spot.